src/database/db.js

```diff
@@ -2,11 +2,8 @@ const typeorm = require('typeorm');
 
 const typeormConnection = new typeorm.DataSource({
     type: 'mysql',
-    host: process.env.SQL_HOST,
-    username: process.env.SQL_USER,
-    database: process.env.SQL_DATABASE,
+    url: process.env.DATABASE_URL,
     port: process.env.SQL_PORT,
```

Comment:
Port can be included in the URL as well so this can be removed.

Reply:
Forgot to remove thank you!